07_memory/usr_mem_test.log

```diff
@@ -0,0 +1,370 @@
+====== Run memory test allocation! ============================
+|
+| function call: malloc
+| test case:     2^x
+|
+===============================================================
+| Id: 0
+| alloc time: 1160(ns)	free time: 548(ns)
```

Comment:
Please print results in one line for each size.

Reply:
Done.